hs_core/hydroshare/resource.py

```diff
@@ -1050,6 +1050,9 @@ def submit_resource_for_review(request, pk):
     resource.raccess.immutable = True
     resource.raccess.save()
 
+    # create review date
+    resource.metadata.create_element('date', type='review_started', start_date=resource.updated)
```

Comment:
I am assuming if the resource metadata review is rejected the first time, the user can try to publish the same resource again. In that case this will raise an exception.

Reply:
really good catch once again Pabitra! Thanks. I believe I resolved...